swarm.sh

```diff
@@ -48,7 +48,12 @@ main() {
     config::await_service_removed instant_message-bus-kafka-config-importer
     config::await_service_removed instant_sante-mpi-config-importer
     config::await_service_removed instant_hapi-fhir-config-importer
-     
+
+    log info "Restarting HAPI FHIR.."
+    try "docker service scale instant_hapi-fhir=0" "Error scaling down hapi-fhir to update the IG"
+    sleep 10
```

Comment:
As far as I know, scaling a service down does a convergence check. Do you get errors if not using this sleep?

Reply:
let me test it, and it works I will push again 👍🏼

Reply:
It works! nice Thanks Mark, I updated that 🚀